package.json

```diff
@@ -48,7 +42,7 @@
     "typescript": "^4.9.5"
   },
   "dependencies": {
-    "@folio/stripes-acq-components": "~4.0.0",
+    "@folio/stripes-acq-components": "~5.0.0",
     "@folio/stripes-data-transfer-components": "^5.4.0",
```

Comment:
folio/stripes-data-transfer-components module has already upgraded react also, upgrade it to v6.0.0 as well

Reply:
Thank you! Done.